guides/documentations/configuration.md

```diff
@@ -0,0 +1,91 @@
+# How to set your plugin's configuration
+
+The DefaultConfiguration class is used to set your plugin's configuration.
+
+In this part, you will see how you can:
+- Tag your plugin
+- Define the type for your plugin diagram
+- Define a default file name and file extension
+- Define a restrictive folder
+- Set the configuration for the code editor
+
+## Tag your plugin
+
+To help users understand the usage of your plugin, you can tag it using two types of tags:
+- `language`: This tag indicates the programming language used in your plugin.
```

Comment:
put in second line to be coherent.

Reply:
done

Reply:
I mean category have to be at first in the `-` list

Reply:
done